### PR TITLE
Hod/mime cast fix

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22214

## Description
Parsing of the create-incident response was broken, see https://www.mimecast.com/tech-connect/documentation/endpoint-reference/threat-intel/create-incident/#response for more details

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py [hunk 1]
```diff
@@ -68,13 +66,7 @@
 
 def determine_ssl_usage():
     global USE_SSL
-
```
Comment (Itay4) [inline Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py:71]: why did you remove that?

File: Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py [hunk 2]
```diff
@@ -1,6 +1,4 @@
-import demistomock as demisto
```
Comment (Itay4) [inline Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py:1]: revert this

File: Packs/Mimecast/pack_metadata.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-    "Email Gateway"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
-}
+    "name": "Mimecast",
+    "description": "Mimecast unified email management offers cloud email services for email security, continuity and archiving emails. Please read detailed instructions in order to understand how to set the integration's parameters.",
+    "support": "xsoar",
+    "currentVersion": "1.1.0",
```
Comment (Itay4) [inline Packs/Mimecast/pack_metadata.json:5]: this should be revision, not minor

File: Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py [hunk 4]
```diff
@@ -2099,71 +2091,74 @@ def get_mimecast_incident_request():
...
 def mimecast_incident_api_response_to_markdown(api_response, action_type):
-    incident_code = api_response['data'][0]['code']
-    incident_type = api_response['data'][0]['type']
-    incident_reason = api_response['data'][0]['reason']
-    incident_identified_messages_amount = api_response['data'][0]['Identified']
-    incident_successful_messages_amount = api_response['data'][0]['Successful']
-    incident_failed_messages_amount = api_response['data'][0]['Failed']
-    incident_restored_messages_amount = api_response['data'][0]['Restored']
-    incident_id = api_response['data'][0]['id']
+    incident_code = api_response['data'][0].get('code', '')
```
Comment (Itay4) [inline Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py:2094]: lets initialize a var and use it instead, like `data = api_response.get('data', [{}])[0]`

**Comment by kirbles19** [comment]
@hod-alpert @Itay4 Done.
